Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1451 @@
+        return self._http_request(method='PUT', url_suffix=url_suffix, json_data=body)
+
+    def pull_request_comment_delete_request(self, repo: str, pr_id: str, comment_id: str) -> Response:
+        """ Deletes a specific comment in a pull request.
+            Args:
+                repo: str - The repository the user entered, if he did.
+                pr_id: str - an id to a specific pull request to add a comment to.
+                comment_id: str - an id to a specific comment to delete.
+            Returns:
+                a Response object.
+        """
+        url_suffix = f'/repositories/{self.workspace}/{repo}/pullrequests/{pr_id}/comments/{comment_id}'
+        return self._http_request(method='DELETE', url_suffix=url_suffix, resp_type='response')
+
+    def workspace_member_list_request(self, page: int, page_size: int) -> Dict:
+        """ Gets a list of the members in the workspace.
+            Args:
+                page: int - The specific result's page.
+                page_size: int - The number of wanted results in a results page.
+            Returns:
+                A response object in a form of a dictionary.
+        """
+        params = {
+            'page': page,
+            'pagelen': page_size
+        }
+        url_suffix = f'/workspaces/{self.workspace}/members'
+        return self._http_request(method='GET', url_suffix=url_suffix, params=params)
+
+
+def get_exception_message(e: DemistoException) -> Dict:
+    """ extract the dictionary with the message about the exception.
+        Args:
+            e: DemistoException - the exception string.
+        Returns:
+            A json object containing the error message.
+    """
+    message_arr = e.message.split('\n')
+    m_json = json.loads(message_arr[1])
+    m_json['message_arr'] = message_arr[0]
+    return m_json
+
+
+def check_pagination(client: Client, response: Dict, limit: int) -> List:
+    """ Check if pagination is needed.
+    Args:
+        client: A Bitbucket client.
+        response: A response from the API call.
+        limit: A limit to the list.
+    Returns:
+        An array of all the relevant results.
+    """
+    arr: List[Dict] = response.get('values', [])
+    is_next = response.get('next', None)
+    pagelen = response.get('pagelen', 50)
+    if is_next and limit > int(pagelen):
+        return get_paged_results(client, response, limit)
+    else:
+        return arr
+
+
+def get_paged_results(client: Client, response: Dict, limit: int) -> List:
+    """ Uses pagination and returns a list of results.
+    Args:
+        client: A Bitbucket client.
+        response: A response from the API call.
+        limit: A limit to the list
+    Returns:
+        Returns a list of results through pagination..
+    """
+    results = []
+    arr: List[Dict] = response.get('values', [])
+    is_next = response.get('next', None)
+    while response:
```

Comment:
Why do we need this while?

Reply:
Because when we have a new response (in line 450), and a list of results (in line 452) ,we want to go back to line 447, and add the new list to the final results list, and check again if there are more results to get from Bitbucket or not. 
Maybe I can change it to while true, if you think it is better.
I think the loop is better than a recursive call with every new response, because of the memory usage in recursion.